tests/unit/artifact_schemas/precision_recall_curve_v0.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$id": "https://amazon.com/sagemaker.graph.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "PrecisionRecallCurve",
+  "type": "object",
+  "description": "Describes serialized precision recall curve in json format.",
+  "properties": {
+    "type": {
+      "description": "Constant value that provides an indicator on how to interpret this data.",
+      "const": "PrecisionRecallCurve"
+    },
+    "version": {
+      "description": "Version of the schema",
+      "const": "0"
+    },
+    "title": {
+      "description": "Title of the plot",
+      "type": "string"
+    },
+    "precision": {
+      "description": "Precision values such that element i is the precision of predictions with score >= thresholds[i] and the last element is 1.",
+      "type": "array"
+    },
+    "recall": {
+      "description": "Decreasing recall values such that element i is the recall of predictions with score >= thresholds[i] and the last element is 0.",
+      "type": "array"
+    },
+    "averagePrecisionScore": {
+      "description": "AP summarizes a precision-recall curve as the weighted mean of precisions achieved at each threshold, with the increase in recall from the previous threshold used as the weight.",
+      "type": "number"
+    }
+  },
+  "additionalProperties": false
+}
```

Comment:
nit: add new line at the end of file, same as a few class below.

Reply:
added